best-practices.md

```diff
@@ -5,6 +5,7 @@
 * [Field and ID formatting](#field-and-id-formatting)
 * [Field selection and Metadata Linking](#field-selection-and-metadata-linking)
 * [Datetime selection](#datetime-selection)
+* [Null Geometries](#null-geometries)
```

Comment:
I think this section name is a bit technical. Would think "Unlocated Items" or so is more intuitive.

Reply:
Good call, will change.